lbrynet/daemon/auth/server.py

```diff
@@ -298,12 +298,13 @@ def _render_error(self, failure, request, id_):
                 # maybe its a twisted Failure with another type of error
                 error = JSONRPCError(failure.getErrorMessage() or failure.type.__name__,
                                      traceback=failure.getTraceback())
+            if not failure.check(ComponentsNotStarted, ComponentStartConditionNotMet):
+                log.warning("error processing api request: %s\ntraceback: %s", error.message,
+                            "\n".join(error.traceback))
         else:
             # last resort, just cast it as a string
             error = JSONRPCError(str(failure))
-        if not failure.check(ComponentsNotStarted, ComponentStartConditionNotMet):
-            log.warning("error processing api request: %s\ntraceback: %s", error.message,
-                        "\n".join(error.traceback))
+
```

Comment:
Above code fails in cases where `failure` is not a `Failure` and thus there is no `.check()` method. Instead it is moved to be inside `if isinstance(failure, Failure)` where `failure.check()` works.

Reply:
@jackrobison ^

Reply:
:+1: